docs/sources/plugins/catalog.md

```diff
@@ -6,17 +6,59 @@ weight = 1
 
 # Plugin catalog
 
-Catalog for Grafana is a plugin bundled with Grafana versions 8.0+. It allows users to browse and manage plugins from within Grafana. Only Grafana Admins can access and use the Marketplace.
+The Plugin catalog, new in Grafana 8.0, allows users to browse and manage plugins from within Grafana. Only Grafana Admins can access and use the catalog.
```

Comment:
```suggestion
The Plugin catalog allows users to browse and manage plugins from within Grafana. Only Organization Admins who are also Grafana Server Admins can access and use the catalog.
```

Reply:
Good point - we should verify and probably add this information as a Note.